documentation/docs/pages/Parameter-choices.md

```diff
@@ -0,0 +1,249 @@
+
+### Thermodynamics and Equation of State (TEOS-10)
+The configuration uses `EQN_OF_STATE = "ROQUET_RHO"` for seawater. [`ROQUET_RHO`](<https://mom6.readthedocs.io/en/main/api/generated/pages/Equation_of_State.html#:~:text=in%20situ%20temperature).-,ROQUET_RHO%20Equation%20of%20State,-%C2%B6>) is based on TEOS-10, but uses a 75-term polynomial to compute in-situ density as a function of conservative temperature and absolute salinity, closely approximating the full TEOS-10 results [@roquet2015accurate]. Prognostic temperature and salinity are conservative temperature and absolute salinity (`USE_CONTEMP_ABSSAL = True`), consistent with the equation of state. However, the cold-start initial conditions in our configuration use an inconsistent temperature–salinity pair - specifically, conservative temperature combined with practical salinity (see [issues/235](https://github.com/ACCESS-NRI/access-om3-configs/issues/235)).
+
+
+The `_RHO` variant specifically fits density rather than specific volume, ideal for layered models and ensuring that neutral density calculations are precise. The freezing conservative temperature is calculated from absolute salinity and pressure using a 23-term polynomial fit refactored from the TEOS-10 package (`TFREEZE_FORM = "TEOS_POLY"`). More relevant discussions or notes can be found in [TWG-23-July-2025](https://forum.access-hive.org.au/t/cosima-twg-meeting-minutes-2025/4067/17#:~:text=Freezing%20temperature%20consistency%20between%20mom6%20and%20cice). 
+
+### Surface freezing and salinity constraints
+At the ocean surface, we have enabled the realistic sea-ice formation and salinity. We've turned on frazil ice formation (`FRAZIL = TRUE`), which means if the ocean surface layer temperature drops below the surface freezing point, the model will convert the negative heat content into frazil ice instead of allowing the water to supercool. By enabling frazil, any heat deficit below freezing in the ocean is passed to the sea-ice component as latent heat of fusion, preventing unrealistically cold liquid water. 
```

Comment:
a small correction
```suggestion
At the ocean surface, we have enabled realistic sea-ice formation and salinity. We've turned on frazil ice formation (`FRAZIL = TRUE`), which works upwards through each water column, transferring heat downwards from the layer above as needed to prevent the in-situ temperature falling below the local freezing point in each layer in turn. If the top layer is below freezing, heat is extracted from the sea ice model, which grows frazil ice in response. More details are [here](https://mom6.readthedocs.io/en/main/api/generated/pages/Frazil_Ice.html).
```